metrics/web/test/base/presentation/graphs/animated_bar_test.dart

```diff
@@ -0,0 +1,76 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/graphs/animated_bar.dart';
+import 'package:rive/rive.dart';
+
+import '../../../test_utils/metrics_themed_testbed.dart';
+
+void main() {
+  group("AnimatedBar", () {
+    testWidgets(
+      "throws an AssertionError if the given width is null",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const _AnimatedBarTestbed(width: null));
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "throws an AssertionError if the given width is negative",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const _AnimatedBarTestbed(width: -1.0));
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+  });
```

Comment:
It seems like we should test that the AnimatedBar applies each of the given parameters.